tests/unit-tests/blocks/test-class-sensei-global-blocks.php

```diff
@@ -64,22 +64,27 @@ public function testEnqueueBlockAssets_WhenCalled_EnqueueBlockAssets() {
 		/* Assert */
 		$this->assertTrue( wp_script_is( 'sensei-course-list-filter' ) );
 		$this->assertTrue( wp_style_is( 'sensei-global-blocks-style' ) );
+
+		/* Reset */
+		wp_dequeue_script( 'sensei-course-list-filter' );
+		wp_dequeue_style( 'sensei-global-blocks-style' );
 	}
 
 	/**
 	 * Enqueue the global block assets, except when the user is on the admin page.
 	 */
 	public function testEnqueueBlockAssets_WhenCalledOnAdmin_NotEnqueueCourseListFilter() {
 		/* Arrange */
-
-		$this->markTestSkipped( 'This test requires WordPress 5.8 or higher.' );
```

Comment:
This test was waiting for a minimum supported WP version higher than 5.8, which we now have.